Formula/nghttp2-libonly.rb

```diff
@@ -0,0 +1,54 @@
+class Nghttp2Libonly < Formula
```

Comment:
We'll not include this formula for now so depending on `nghttp2` would be the way forward despite the boost dep. Can you switch it back ?

Reply:
Sure. It was getting pulled in by httpd anyway, although it too could benefit from a lighter nghttp2.